apps/web/components/apps/AdditionalCalendarSelector.tsx

```diff
@@ -2,78 +2,71 @@ import type { FunctionComponent, SVGProps } from "react";
 
 import { InstallAppButton } from "@calcom/app-store/components";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
-import { QueryCell } from "@calcom/trpc/components/QueryCell";
 import { trpc } from "@calcom/trpc/react";
+import { Button } from "@calcom/ui/components/button";
```

Comment:
Just to understand the purpose, is this import instead of "@calcom/ui" because of TS perf issue? or is there any other reason?

Reply:
This is not strictly necessary, but it does improve performance, when you include a file like "@calcom/ui" ts evaluates all files referenced within that file, rather than just in Button. so if you're more specific, performance improves.